### PR TITLE
Remove unnecessary Newtonsoft

### PR DESCRIPTION
**Pull request by SanSYS**
I have a little problem - big solution, with references to Newtonsoft Json 9 version
And I want to use your library
But reference to Newtonsoft version of 11 breaks my project
After review your code I understood, that Newtonsoft used only for tracing to console
Not for production ;)

So.. I created interface IRequestLogger with default implemetation via StringBuilder and trace to log raw HTTP request. It's very useful for debug)
And if anybody wants to use newtonsoft, csv or yet another logger - he could create himself implemetation )

Demo from examples with turned on tracing https://gist.github.com/SanSYS/2b9c2c57c2698e261a8a3bc127508df1

### CODE REVIEW

**Comment by SanSYS** [comment]
@poornas, I promised one commit, but I'm really don't know what to do with merge commit (

File: Minio.Examples/Program.cs [hunk 1]
```diff
@@ -184,10 +184,13 @@ public static void Main(string[] args)
                 // Delete the object
                 Cases.RemoveObject.Run(minioClient, destBucketName, objectName).Wait();
 
+                // Tacing request with custom logger
+                Cases.CustomRequestLogger.Run(minioClient).Wait();
```
Comment (poornas) [inline Minio.Examples/Program.cs]: This example can be commented out so that output is clean
s/Tacing/Tracing

**Comment by poornas** [comment]
> @poornas, I promised one commit, but I'm really don't know what to do with merge commit (
> 
@SanSYS , that's ok, we can squash on merging

File: Minio.Examples/Program.cs (hunk 1 above)
Comment (SanSYS) [reply Minio.Examples/Program.cs]: Sorry, I didn't see this comment early

**Comment by SanSYS** [comment]
Do you know next release date?

**Comment by poornas** [comment]
@SanSYS , we can make a release today

**Pull request merged**